File: CBASIC/UtilFiles/RecipieManager.js
// Simple Recipe Manager

function recipeManager() {
    const recipeName = prompt("Enter the name of the recipe:");
    const numIngredients = parseInt(prompt("Enter the number of ingredients:"));
    const ingredients = [];

    for (let i = 0; i < numIngredients; i++) {
        let ingredient = prompt(`Enter ingredient ${i + 1}:`);
        ingredients.push(ingredient);
    }

    const instructions = prompt("Enter cooking instructions:");

    // Display the recipe
    console.log(`\nRecipe for: ${recipeName}`);
    console.log("Ingredients:");
    ingredients.forEach((ingredient) => {
        console.log(`- ${ingredient}`);
    });
    console.log(`Instructions: ${instructions}`);
}

// Run the recipe manager
recipeManager();

File: CBASIC/UtilFiles/RecipieManager.py
# Simple Recipe Manager

def recipe_manager():
    recipe_name = input("Enter the name of the recipe: ")
    num_ingredients = int(input("Enter the number of ingredients: "))
    ingredients = []

    for i in range(num_ingredients):
        ingredient = input(f"Enter ingredient {i + 1}: ")
        ingredients.append(ingredient)

    instructions = input("Enter cooking instructions: ")

    # Display the recipe
    print(f"\nRecipe for: {recipe_name}")
    print("Ingredients:")
    for ingredient in ingredients:
        print(f"- {ingredient}")
    
    print(f"Instructions: {instructions}")

# Run the recipe manager
recipe_manager()
